roles/openstack_helm_designate/defaults/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# .. Copyright (C) 2022 VEXXHOST, Inc.
+# .. SPDX-License-Identifier: Apache-2.0
+
+# Default variables
+# =================
+
+# .. contents:: Sections
+#    :local:
+
+
+# .. envvar:: openstack_helm_designate_chart_repo_name [[[
+#
+# Helm repository name for the chart.
+openstack_helm_designate_chart_repo_name: openstack-helm
+
+                                                                   # ]]]
+# .. envvar:: openstack_helm_designate_chart_repo_url [[[
+#
+# Helm repository URL for the chart.
+openstack_helm_designate_chart_repo_url: https://tarballs.opendev.org/openstack/openstack-helm/
+
+                                                                   # ]]]
+# .. envvar:: openstack_helm_designate_chart_name [[[
+#
+# Helm chart name (will also be used for release name)
+openstack_helm_designate_chart_name: designate
+
+                                                                   # ]]]
+# .. envvar:: openstack_helm_designate_image_repository [[[
+#
+# Image repository location to be prefixed for all images
+openstack_helm_designate_image_repository: "{{ atmosphere_image_repository | default('us-docker.pkg.dev/vexxhost-infra/openstack') }}"
+
+                                                                   # ]]]
+# .. envvar:: openstack_helm_designate_image_tag [[[
+#
+# Image tag for container
+openstack_helm_designate_image_tag: 0.2.7
```

Comment:
I suggest that you update this to say `wallaby` instead, since that will be the newest promoted build.